File: poo/java/Professor/ProfessorIntegral.java
/*
 *  Programa escrito com propósito didático apenas.
 *  @author Nathan Cruz <nathan.cruz@unifesp.br>
 *  @created 04/04/14
 *  R.A.: 86492
 */

public class ProfessorIntegral extends Professor{
    
    /* Atributos */
    private float salario;

    /* Construtor */
    public ProfessorIntegral(String nome, int idade, 
                             int matricula, float salario){
        super(nome, idade, matricula);
        this.salario = salario;
    }

    /* Getters */
    public float getSalario(){
        return this.salario;
    }

    /* Setters */
    public void setSalario(float salario){
        this.salario = salario;
    }

    @Override
    /* Imprime os atributos */
    public void Imprime(){
        super.Imprime();
        System.out.println("\tSalário:\t" + this.getSalario());
    }

}

File: poo/python/Professor/ProfessorIntegral.py
#!/usr/bin/env python
#-*- coding: utf-8 -*-

"""
@author Nathan Cruz <nathan.cruz@unifesp.br>
@created 06/04/14
"""
from __future__ import unicode_literals
from __future__ import print_function

from Professor import Professor

class ProfessorIntegral(Professor):

    # Atributos #
    __salario = None

    # Construtor #
    def __init__(self, nome, idade, matricula, salario):
        super(ProfessorIntegral, self).__init__(nome, idade, matricula)
        self.__salario = salario

    # Getters #
    def getSalario(self):
        return self.__salario

    # Setters #
    def setSalario(self, salario):
        self.__salario = salario

    # Redefinido #
    # imprime os atributos #
    def imprime(self):
        super.imprime()
        print('\tSalário:\t%.2f' % (self.getSalario()))
